tests/classify_test.py

```diff
@@ -128,3 +128,12 @@ def test_application_directories(in_tmpdir, no_empty_path):
     # Should be application with extra directories
     ret = classify_import('testing', application_directories=('.', 'tests'))
     assert ret is ImportType.APPLICATION
+
+
+def test_application_modules(in_tmpdir, no_empty_path):
+    # Should be classified 3rd party without argument
+    ret = classify_import('testing')
```

Comment:
let's use some name other than testing like `c_module` to better demonstrate what this is intended for -- `testing` is a common application module name that I use for not-packaged-but-not-tests code

Reply:
Done.